multicluster/test/integration/README.md

```diff
@@ -0,0 +1,29 @@
+# Integration Test Framework
+
+The integration test framework for Antrea multi-cluster is mainly using [envtest](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/envtest)
+which is provided by `controller-runtime`. There is an online book
+about kubebuilder mentioned a few steps about [how to configure envtest](https://book.kubebuilder.io/reference/envtest.html)
+for integration tests. In Antrea multi-cluster, we are using existing clusters
+instead of `kubectl`,`api-server` and `etcd` binaries to run integration test.
+
+# Running the Antrea Multi-cluster Integration tests
+
+The tests must be run on an real Kubernetes cluster. At the moment, you can
+simply run `make test-integration` in `antrea/multicluster` folder. it will
+create a kind cluster named `antrea-integration-kind` and executes the integration
+codes.
+
+if you'd like to run the integration test in an existing Kubernetes cluster, you
+can save your kubeconfig file as `mc-integration-kubeconfig` in `/tmp` directly,
+the integration test codes will read the file `/tmp/mc-integration-kubeconfig` by
+default, then run below commands:
+
+```bash
+export KUBECONFIG=/tmp/mc-integration-kubeconfig
+kubectl create namespace leader-ns
+kubectl apply -f test/integration/cluster-admin.yml
+go test -coverpkg=antrea.io/antrea/multicluster/controllers/multicluster/... -coverprofile=../.coverage/coverage-integration.txt -covermode=atomic -cover antrea.io/antrea/multicluster/test/integration/...
+```
+
+you should be able to see the overall results from terminal, if you'd like to
+check more details, you can go check the file `.coverage/coverage-integration.txt`.
```

Comment:
Can we add section to briefly describe what tests we have? We can add comments in each test file too.

Reply:
done, adding briefly description on the top of each file.